tests/rules/prefer-theme-color-classnames.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { RuleTester } from 'eslint';
+
+import { preferThemeColorClassnames } from '../../src/rules/prefer-theme-color-classnames'; // Adjust the path as necessary
```

Comment:
Remove this comment
```suggestion
import { preferThemeColorClassnames } from '../../src/rules/prefer-theme-color-classnames';
```

Reply:
Updated https://github.com/MetaMask/eslint-plugin-design-tokens/pull/22/commits/8f3d0fb3fc448d4f0c1df4a8a6b385d5f69a05c1